cmd/antrea-agent/options.go

```diff
@@ -37,6 +37,9 @@ const (
 	defaultMTUGRE             = 1462
 	defaultMTUSTT             = 1500
 	defaultMTU                = 1500
+	// IPsec ESP can add a maximum of 38 bytes to the packet including the ESP
+	// header and trailer.
+	ipsecESPOverhead = 38
```

Comment:
I still feel like we are double-counting the outer IP header here (once in `defaultMTUGRE` and once in `ipsecESPOverhead`) but I'm not that familiar with IPsec.

Reply:
No IPsec is a new tunnel and adds a new IP header. It does not reuse the GRE tunnel IP header.

Reply:
my bad, didn't know about that